src/platform/packages/shared/kbn-scout/src/config/stateful/base.config.ts

```diff
@@ -178,6 +178,42 @@ export const defaultConfig: ScoutServerConfig = {
       '--xpack.ruleRegistry.write.enabled=true',
       '--xpack.ruleRegistry.write.cache.enabled=false',
       '--monitoring_collection.opentelemetry.metrics.prometheus.enabled=true',
+      '--xpack.profiling.enabled=true',
```

Comment:
Does it mean Profiling is enabled on stateful by default? Can we run the tests against ECH and expect it to pass?

Reply:
I don't believe it's enabled by default, so this may also be a problem. I'll do some investigation on a cloud instance, and see if I can get it working.